platform/landing-page/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -55,6 +55,11 @@
                 name="Branding Logo" id="logo" required="true" type="String"
                 default=""
                 />
+        <AD
+                description="Additional links to be displayed on the landing page. Use the format <text>,<link>. For example: connexta, http://www.connexta.com"
```

Comment:
Per @brendan-hofmann's request: send me the documentation updates as well, as the landing page PR for the draft-documentation edit has not yet merged.